seatunnel-api/src/main/java/org/apache/seatunnel/api/sink/SinkWriter.java

```diff
@@ -92,6 +92,11 @@ interface Context extends Serializable {
         /** @return The index of this subtask. */
         int getIndexOfSubtask();
 
+        /** @return parallelism of this writer. */
+        default int getNumberOfParallelSubtasks() {
```

Comment:
I am looking for a way to get spark/flink to support both returning parallelism.